cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,37 @@
 /// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
 
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.viewport(1200, 1200)
+    cy.visit('https://demoqa.com/automation-practice-form')
   });
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.firstName();
```

Comment:
maybe it would be better to use lastName after faker.name

Reply:
true